src/deployment/deploy.py

```diff
@@ -426,7 +426,16 @@ def try_sp_create() -> None:
             }
 
         else:
+            onefuzz_cli_app = cli_app[0].app_id
             authorize_application(uuid.UUID(ONEFUZZ_CLI_APP), app.app_id)
```

Comment:
my bad there was a error in the code suggested
```suggestion 
onefuzz_cli_app = cli_app[0]
authorize_application(uuid.UUID(onefuzz_cli_app.app_id), app.app_id)
```